noxfiles/docker_nox.py

```diff
@@ -2,6 +2,7 @@
 import nox
 from constants_nox import (
     IMAGE,
+    IMAGE_DEV,
```

Comment:
It won't let me mark it because nothing has changed there, but should `dev` in `build_matrix` be `IMAGE_DEV`?

```diff
- build_matrix = {
-     "prod": {"tag": get_current_image, "target": "prod"},
-     "dev": {"tag": lambda: IMAGE_LOCAL, "target": "dev"},
-     "test": {"tag": lambda: IMAGE_LOCAL, "target": "prod"},
-     "ui": {"tag": lambda: IMAGE_LOCAL_UI, "target": "frontend"},
- }
+ build_matrix = {
+     "prod": {"tag": get_current_image, "target": "prod"},
+     "dev": {"tag": lambda: IMAGE_DEV, "target": "dev"},
+     "test": {"tag": lambda: IMAGE_LOCAL, "target": "prod"},
+     "ui": {"tag": lambda: IMAGE_LOCAL_UI, "target": "frontend"},
+ }
```

Reply:
no, the setup is slightly complex but if you check `docker-compose.yml` the dev image has to match the image it is expecting there (tagged as `local`). The difference is just the target stage of the docker build, it should use the same tag